ci/debian/centreon-connector-ssh.install

```diff
@@ -1 +1 @@
-debian/tmp-centreon-collect/usr/lib64/centreon-connector/centreon_connector_ssh usr/lib64/centreon-connector
+debian/tmp-centreon-collect/usr/bin/centreon_connector_ssh usr/bin
```

Comment:
why ?

Reply:
right location after build and destination like rpm package (/usr/bin)

Reply:
on rpm package it's /usr/lib64/centreon-connector/

Reply:
Same remark here